presets/sysadmin.sh

```
@@ -0,0 +1,3 @@
+PRESET_NAME="SysAdmin"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)